README.md

```diff
@@ -50,6 +50,29 @@ To get the config & update eve image stored in dist/amd64/installer/rootfs.img
  eden controller -m adam:// edge-node eveimage-update dist/amd64/installer/rootfs.img
 ``` 
 
+## Connect to eve console
+
+``` 
+eden eve console
+``` 
+
+## Run applications on EVE
+
+Deploy nginx server from dockerhub. Expose port 80 of the container to port 8081 of eve. 
+``` 
+eden pod deploy -p 8081:80 docker://nginx
```

Comment:
It is not working example for Qemu (please use another external port, forwarded by qemu, 8027/tcp or 8028/tcp)
You can find defaults ports here: https://github.com/itmo-eve/eden/blob/18b6e56978f85648f5a7df7ced502111dd27ff6b/pkg/utils/config.go#L173-L178